components/LargeInfoSection.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useEffect } from 'react';
+import { motion, useAnimation } from 'framer-motion';
+import { useContextTheme } from './ThemeContext';
+import { useInView } from 'react-intersection-observer';
+import useStripeSession from '../hooks/useStripeSession';
+
+function LargeInfoSection({ isLeftLayout }) {
+  const animation = useAnimation();
+  const [handleSubmit, isPending] = useStripeSession();
+  const { ref, inView } = useInView({ threshold: 0.4 });
+  const { cardsBackgroundColor, theme } = useContextTheme();
+
+  useEffect(() => {
+    const isMobile = window.innerWidth < 768;
```

Comment:
Maybe we can use <style jsx> and use media queries instead of running a useEffect with an external library.